R/get_meetnetten_locations.R

```diff
@@ -0,0 +1,178 @@
+#' within a specified species group. When no monitoring scheme or species group
+#' is specified, the observations of all monitoring schemes are returned.
+#'
+#' @param scheme_name the name of the monitoring scheme for which you want to
+#' extract location data. Data from multiple schemes can be selected by providing
+#' a vector with the names of the schemes.
+#' @param species_group the name of the species group for which you want to
+#' extract location data. Data from multiple species groups can be selected by
+#' providing a vector with the names of the species groups.
+#' @param connection dbconnection with the database 'S0008_00_Meetnetten'
+#' on the inbo-sql08-prd.inbo.be server
+#'
+#' @return A list with two sf objects:
+#'
+#' \itemize{
+#'  \item \code{main_locations}: sf object of the main locations of the
+#'    selected monitoring schemes, with following attribute variables:
+#'    \itemize{
+#'    \item \code{species_group}
+#'    \item \code{scheme}: name of the monitoring scheme
+#'    \item \code{location}: name of the location
+#'    \item \code{is_sample}: whether the location belongs to the sample of
+#'    locations for the monitoring scheme (see details)
+#'    \item \code{is_active}: when a location is not suited for counting any
+#'    more, the location becomes inactive (\code{is_active} = \code{FALSE})
+#'    }
+#' \item \code{sublocations}: sf object of the the sublocations (for example
+#' the sections of a transect) for each of the selected main locations, with
+#' following attribute variables:
+#'    \itemize{
+#'    \item \code{species_group}
+#'    \item \code{scheme}: name of the monitoring scheme
+#'    \item \code{location}: name of the main location
+#'    \item \code{sublocation}: name of the sublocation
+#'    \item \code{is_active}: whether the sublocation is counted or not
+#'    }
+#' }
+#'
+#' Not all main locations are subdivided in sublocations.
+#' So in some cases the sublocations object is empty.
+#'
+#' @details
+#' Each monitoring scheme of the species monitoring programme of Flanders
+#' \href{www.meetnetten.be}{Meetnetten} consists of a fixed set of locations.
+#' A monitoring scheme for rare species includes all locations where the species
+#' occurs. For more common species a sample of locations is drawn and the
+#' the selected locations are included in the monitoring scheme. In some cases,
+#' the monitoring project in \href{www.meetnetten.be}{Meetnetten} also contains
+#' locations that are not part of the sample. These locations can be counted
+#' optionally and are indicated by (\code{is_sample} = \code{FALSE}).
+#'
+#' It also occurs that a location becomes inaccessible or that the target
+#' species disappears. Then, a locations can be made inactive (\code{is_active}
+#' = \code{FALSE}), which means that no observations can be recorded any more.
+#'
+#' @importFrom assertthat assert_that
+#' @importFrom dplyr filter arrange collect tbl sql mutate select
```

Comment:
import all functions. You're missing %>%, .data, ...

Reply:
OK